aspnet/hosting/aspnet-core-module.rst

```diff
@@ -0,0 +1,155 @@
+|                           | |                                                  |
+|                           | | If  true, the token will be forwarded to the     |
+|                           | | child process listening on %HTTP_PLATFORM_PORT%  |
+|                           | | as a header 'X-IIS-WindowsAuthToken' per request.|
+|                           | | It is the responsibility of that process to call |
+|                           | | CloseHandle on this token per request.           |
+|                           | |                                                  |
+|                           | | The default value is false.                      |
++---------------------------+----------------------------------------------------+
+
+Child Elements
+^^^^^^^^^^^^^^^
+
++---------------------------+----------------------------------------------------+
+| Attribute                 |     Description                                    |
++===========================+====================================================+
+| environmentVariables      | | Configures **environmentVariables** collection   |
+|                           | | for the process specified in **processPath**.    |
++---------------------------+----------------------------------------------------+
+
+ASP.NET Core Module app offline
+-------------------------------
+
+If you place a file with the name  *app_offline.htm* at the root of a web application directory, the ASP.NET Core Module will shut-down the application and stop processing any new incoming requests for that application. The ASP.NET Core Module will respond to all requests for dynamic pages in the application by sending back the content of the  *app_offline.htm*  file. For example, you might want to return a "site under construction" or a "down for maintenance" page.
+
+Once the *app_offline.htm* file is removed, the next request will load the application and the application will respond to requests.
+
+
+ASP.NET Core Module configuration examples
+------------------------------------------
+
+.. _log-redirection:
+
+Log creation and redirection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For logs to be saved, you must create the log directory. The ASP.NET Core Module can redirect ``stdout`` and ``stderr`` logs to disk by setting the ``stdoutLogEnabled``  and ``stdoutLogFile`` properties of the ``aspNetCore`` attribute. Logs are not rotated. It is the responsibilty of the hoster to limit the disk space the logs consume.
+
+.. literalinclude:: aspnet-core-module/sample/web.config
+  :language: xml
+  :lines: 12-16,20
+
+
+Setting environment variables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ASP.NET Core Module allows you specify environment variables for the process specified in the ``processPath`` setting by specifying them in ``environmentVariables`` child attribute to the ``aspNetCore`` attribute. The example below illustrates how you would use it.
```

Comment:
I think these are both elements.

> specifying them in ``environmentVariables`` child element to the ``aspNetCore`` element.

... and this might not be necessary at all ...

> The example below illustrates how you would use it.